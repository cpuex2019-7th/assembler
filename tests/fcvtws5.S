    li t1, 1079613850           ; 3.4
    fmvwx ft1, t1

    fcvtws a0, ft1
