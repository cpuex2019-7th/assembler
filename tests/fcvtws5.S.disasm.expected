lui t1, 263578
addi t1, t1, -1638
fmvwx ft1, t1
fcvtws a0, ft1
